### DISCUSSION TITLE
Do I need manually re-import the Key Vault certificate into the App Service or not?

### DISCUSSION THREAD

**Discussion by daiplusplus** [polymind-inc/acmebot#465] [Q&A] [answered]
When using [keyvault-acmebot](https://github.com/shibayan/keyvault-acmebot) (instead of [appservice-acmebot](https://github.com/shibayan/appservice-acmebot)) with Azure App Services, I see that I need to go (App Service) > TLS/SSL Settings > Private Key Certificates > Import Key Vault certificate.

The drop-down list shows the thumbprint of the certificate, suggesting that this import procedure will need to be repeated every time the certificate's thumbprint changes (i.e. every time the certificate is renewed).

...or is it re-imported automagically?

**Comment by shibayan** [comment, marked as answer]
>If you update your certificate in Key Vault with a new certificate, App Service automatically syncs your certificate within 24 hours.

https://docs.microsoft.com/en-us/azure/app-service/configure-ssl-certificate?tabs=apex%2Cportal#import-a-certificate-from-key-vault
